scripts/build/termux_step_make_install.sh

```diff
@@ -3,7 +3,11 @@ termux_step_make_install() {
 	[ "$TERMUX_PKG_METAPACKAGE" = "true" ] && return
 
 	if test -f build.ninja; then
-		ninja -w dupbuild=warn -j $TERMUX_MAKE_PROCESSES install
+		if [ $(echo -e "$(ninja --version)\n1.12.0"|sort -V|head -1) == "1.12.0" ]; then
```

Comment:
Use `dpkg --compare-versions` instead. I dont think that works for anything above 1.12.0.

Reply:
`dpkg --compore-versions` will not work for `termux-pacman` I suppose.

CC: @Maxython

Reply:
Is ninja version check really necessary here? Older ninja versions are not used.

Reply:
> Is ninja version check really necessary here? Older ninja versions are not used.

See https://github.com/termux/termux-packages/blob/3c658f5509ca78161d2da4a63a52b143b56f2ba6/scripts/build/setup/termux_setup_ninja.sh#L2

I'd like to just remove `-w dupbuild=warn`.

Reply:
> `dpkg --compore-versions` will not work for termux-pacman I suppose

Yes, you are right

> There's the equivalent vercmp utility for pacman.
That is part of the pacman package, though I'm not sure if it is being built in Termux.

Don't worry, there is a `vercmp` command in the `pacman` package.

Reply:
How about upgrading njnja to 1.12.0? #19887

Reply:
I think just help update termux_setup_ninja.sh to 1.12.0 instead. No need for version checking. We are going to switch to that version anyway.